net/net/inc/TSecContext.h

```diff
@@ -27,7 +27,7 @@
 #include "TDatime.h"
 
 // Jan 1, 1995, 00:00:00 in sec from EPOCH (Jan 1, 1970)
-const TDatime kROOTTZERO = 788914800;
+constexpr Int_t kROOTTZERO = 788914800;
```

Comment:
You shouldn't change the type of this variable; this changes its meaning and this variable is public / part of the interface.

Instead, please mark it as `extern const` and provide the definition in the source file.